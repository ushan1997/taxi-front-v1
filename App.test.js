import React from 'react';
import  renderer  from 'react-test-renderer'

test('view vehicles in db',()=>{
    const component = renderer.create(<vehicles/>);
    const tree =component.toJSON();
    expect(tree).toMatchSnapshot();
})

