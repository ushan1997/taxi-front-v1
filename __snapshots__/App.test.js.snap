// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`view vehicles in db 1`] = `<vehicles />`;
